test/clj/game/core/say_test.clj

```diff
@@ -303,6 +303,47 @@
         (core/command-parser state :corp {:user user :text "/trace 99999999999999999999999999999999999999999999"})
         (is (= 1000 (:base (prompt-map :corp))) "Base trace should now be 1000"))))
 
+  (testing "/undo-click"
+    (let [r {:username "Runner"}
+          c {:username "Corp"}]
+      (do-game
+        (new-game {:corp {:hand ["Hedge Fund"]}})
+        (dotimes [n 3]
+          (click-credit state :corp))
+        (end-turn state :corp)
+        (start-turn state :runner)
+        (is (changed? [(:click (get-runner)) -1
+                       (:credit (get-runner)) +1]
+              (click-credit state :runner))
+            "Clicked for a cred")
+        (core/command-parser state :runner {:user r :text "/undo-click"})
+        (core/command-parser state :runner {:user r :text "/undo-click"})
```

Comment:
Yeah - previously you could undo-click into your opponents turn. This will catch if we make any changes in the future  that allow that, or allow undo-ing into your pre-draw step. I changed the test to make this more explicit.